app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
```

Comment:
Use [`sanitize`](http://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize) on the user-supplied `params[:query]` here to protect against XSS.